### PR TITLE
Create i have corona page

### PR DESCRIPTION
**Pull request by Nickerty**
Created the "I have Corona Page", which is still fairly basic and needs added functionality, but this is a start.

### CODE REVIEW

File: src/app/i-have-corona/checklist/checklist.component.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <input type = "checkbox">
+    <label> Kortademigheid of ademhalingsproblemen</label><br>
+    <input type = "checkbox">
+    <label> Pijn of druk op de borst</label><br>
+    <input type = "checkbox">
+    <label> Aangetast spraak- of bewegingsvermogen</label><br>
+  </form>
+  <br>
+  <button (click)="showCoronaConfirmed()">Bevestigen</button>
+</div>
```
Comment (annedeg) [inline src/app/i-have-corona/checklist/checklist.component.html:40]: We use angular.
You should put these variables in a list (in a .ts file) and use something like this:
`<form *ngFor='let symptom of symptoms'>`

File: src/app/i-have-corona/checklist/checklist.component.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<div *ngIf="!buttonClicked">
```
Comment (annedeg) [inline src/app/i-have-corona/checklist/checklist.component.html:1]: Nice 👍

File: src/app/i-have-corona/checklist/checklist.component.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  showSymptomsList() {
+    this.buttonClicked = true;
+    this.symptomsPage = true;
+  }
+
+
+
+
+
```
Comment (annedeg) [inline src/app/i-have-corona/checklist/checklist.component.ts:34]: Too many empty lines...

File: src/app/i-have-corona/checklist/checklist.component.html (hunk 2 above)
Comment (Nickerty) [reply src/app/i-have-corona/checklist/checklist.component.html:1]: Thank you

**Pull request merged**